guides/index.html

```diff
@@ -0,0 +1,11 @@
+---
+title: Guides
+---
+
+{% assign pages = site.pages | sort: "title" %}
+
+<ul>
+  {% for p in pages %}{% if p.path contains 'guides/' and p.path != page.path %}
+  <li><a href="{{ p.url}}">{{ p.title }}</a></li>
```

Comment:
Should the lack of a space before the first set of closing braces have been caught by the various linters or formatters?

Reply:
The linter doesn't actually understand Liquid (Jekyll's templating language), but it's HTML compliant so it does the right thing most of the time.